src/sentry/api/endpoints/project_rule_preview.py

```diff
@@ -28,24 +30,32 @@ def post(self, request: Request, project) -> Response:
                 "actionMatch": "all",
                 "filterMatch": "all",
                 "frequency": 60,
+                "endpoint": datetime or None
             }}
 
         """
-        serializer = RuleSetSerializer(
+        if not features.has(
+            "organizations:issue-alert-preview", project.organization, actor=request.user
+        ):
+            return Response(status=404)
+        serializer = RulePreviewSerializer(
             context={"project": project, "organization": project.organization}, data=request.data
         )
 
         if not serializer.is_valid():
             raise ValidationError
 
         data = serializer.validated_data
+        if data.get("endpoint") is None:
```

Comment:
two options here:
1. just put `timezone.now()` as the default for a 1 liner
2. change the condition to `if not` so we can handle empty strings a well

Reply:
Actually now that I've added this to the serializer I think endpoint can only be `None`, not exist, or a valid `datetime`

Reply:
tight ok. actually no need to change to a 1 liner this is fine